doc/whatsnew/2/2.15/index.rst

```diff
@@ -30,6 +30,10 @@ Extensions
 False positives fixed
 =====================
 
+* The ``differing-param-doc`` check was triggered by positional only arguments.
```

Comment:
@Pierre-Sassoulas I didn't notice this, but the changelog entry is in the wrong place. Should be `2.14.4`.

Reply:
I'm going to fix in the maintenance branch at release time.